contrib/codegen-tools/codegen/src/main/ops/org/nd4j/codegen/ops/SDBaseOps.kt

```diff
@@ -1420,6 +1529,36 @@ fun SDBaseOps() =  Namespace("BaseOps"){
         }
     }
 
+
+    Op("stridedSlice") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.shape"
```

Comment:
@treo new op: just exposing what's already there. A minor pain was the default values for the masks but I got that worked out.